### DISCUSSION TITLE
Pouch cell model

### DISCUSSION THREAD

**Discussion by MatHunt** [pybamm-team/PyBaMM#2509] [Q&A]
Is the asymptotic pouch cell model available on Pybamm?

**Comment by rtimms** [comment]
Yes, see [this example ](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/models/pouch-cell-model.ipynb).

The results from the paper are [here](https://github.com/rtimms/asymptotic-pouch-cell).
